api/net/Areas/Editor/Models/Storage/ItemModel.cs

```diff
@@ -32,6 +33,12 @@ public class ItemModel
     /// get/set - The media-type of the file.
     /// </summary>
     public string? MimeType { get; set; }
+
+    /// <summary>
+    /// get/set - The media-type of the file.
+    /// </summary>
+    public string? Modified { get; set; }
```

Comment:
Wouldn't it be better to return a Date instead of a string?  The API is for data, presentation is done by the UI not the API.

Reply:
Fixed.